mandatory/3-function-output.js

```diff
@@ -2,13 +2,16 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+//the getRandomNumber function will get a random number and multipy it by ten
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
+// the combine2words function will take the words and return a new string based on the two words
 
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord.concat(secondWord, thirdWord);
```

Comment:
Nice work. Here you can add blank space so it can pass the test. In the test line there is one space between the words. So if you add space in _firstWord.concat(" ", secondWord, " ", thirdWord)_  it will pass the tests.